rest_auth/serializers.py

```diff
@@ -126,7 +126,7 @@ class UserDetailsSerializer(serializers.ModelSerializer):
     """
     class Meta:
         model = UserModel
-        fields = ('username', 'email', 'first_name', 'last_name')
+        fields = ('id', 'username', 'email', 'first_name', 'last_name')
```

Comment:
@briva I think returning `pk` instead would be better. `id` field is not always the primary key.